src/base/searchengine.h

```diff
@@ -38,10 +40,119 @@
 class QProcess;
 class QTimer;
 
+template <class T, std::size_t N>
+class Version
```

Comment:
No, it does not. You just need to change `operator QString()`, but I think that for the plugin versions we shall not do the omitting.